recipe_batches/recipe_batches.py

```diff
@@ -2,13 +2,47 @@
 
 import math
 
+'''
+Your function should output the maximum number of whole batches that can be made for the supplied recipe using the ingredients available to you, as indicated by the second dictionary.
+
+Look at ingredients on hand
+Compare available amount of ingredients to recipe
+how many times will recipe amount go into available ingredients
+return the amount of times
+
+def rec_batches(rec, ing):
+    loop through ingredients on hand and recipe at same time comparing values
+    keep track of how many times avail ingreds value goes into recipe value
+    what is the largest number that EACH ingredient can go into rec
+
+    ie milk 1288 // 100 = 12 but butter 0 // 5 = 0 so no batches can be made
+'''
+
+rec = {'milk': 2}
+ingreds = {'milk': 200}
+
+
+# for key, value in rec.items():
+#     print(f"{key}: {value}")
+
+# for key, value in ingreds.items():
+#     print(f"{key}: {value}")
+
+
 def recipe_batches(recipe, ingredients):
-  pass 
+    if len(ingredients) < len(recipe):
+        return 0
+    else:
+        for key in recipe and ingredients:
+            return min([ingredients[key] // recipe[key]])
```

Comment:
very nice shorthand use of the Math library. However keep in mind since you are calling `return`, this will only iterate once through the `for` loop. This would be the same as just saying `return min(ingredients['milk'] // recipe['milk'])` without the `for` loop.